events.html

```diff
@@ -137,23 +137,64 @@ <h1 class="title">
                     <div class="field">
                         <div class="control has-icons-left has-icons-right">
                             <input id="game-desc" class="input is-medium" type="text" placeholder="Small">
-                            
                             <span class="icon is-right">
                                 <i class="fas fa-check fa-sm"></i>
                             </span>
                         </div>
                     </div>
-
+                   <!-- date select -->
                     <div class="field">
                         <div class="control has-icons-left has-icons-right">
                             <input id="game-date" class="input is-medium" type="date" placeholder="Normal">
-                            
                             <span class="icon is-medium is-right">
                                 <i class="fas fa-check"></i>
                             </span>
                         </div>
+                        </div>
+
+                        <!-- Hours -->
+                        <div class="field">
+                            <div class="control has-icons-left is-pulled-left">
+                              <div class="select is-success">
+                                <select>
+                                  <option selected>Hour</option>
+                                  <option>1</option>
+                                  <option>2</option>
+                                  <option>3</option>
+                                  <option>4</option>
+                                  <option>5</option>
+                                  <option>6</option>
+                                  <option>7</option>
+                                  <option>8</option>
+                                  <option>9</option>
+                                  <option>10</option>
+                                  <option>11</option>
+                                  <option>12</option>
+                                </select>
+                                <div class="icon is-left">
+                                    <i class="far fa-clock"></i>
+                                  </div>
+                              </div>
+
+                              <!-- Minutes -->
+                              <input class="is-success minutes-input" type="number" placeholder="min" id="minutes" name="quantity" min="1" max="60">
```

Comment:
We'll nee to change this to start at 0 and end at 59, could the single digit numbers also have a 0 in front of them?